basxconnect/core/wizards/add_person.py

```diff
@@ -31,6 +31,8 @@
     PersonAssociation: hg.DIV(layout.form.FormField("name")),
 }
 ADD_ADDRESS_LAYOUT = layout.grid.Grid(
+    layout.grid.Row(layout.grid.Col(_("E-mail Address"))),
+    layout.grid.Row(layout.form.FormField("email"), style="margin-bottom: 4rem"),
```

Comment:
Can you remove this from the PR please? I think it might throw an error.

Reply:
Oh, I forgot to remove it as I was working with the E-mail field at that time.